.vscode/settings.json

```diff
@@ -119,5 +125,7 @@
     
     ],
     "cmake.buildDirectory": "${workspaceFolder}/build-${buildKitVendor}${buildKitVersionMajor}-${variant:cpp}-${variant:toolboxes}-${variant:mor}-${variant:buildType}-${variant:webserver}-${variant:doxygen}-${variant:matplot}",
-    "cmake.parallelJobs": 5
-}
\ No newline at end of file
+    "cmake.parallelJobs": 5,
```

Comment:
I think I have pussed this by error : I think it was `25` before (too much for my personal laptop), I thought I updated this value only locally, but it seems that it commited anyway, sorry about that.
Shall we change it back ?